src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/StateFlowEngine.java

```diff
@@ -233,8 +236,16 @@ public StateFlow build(FlowState.Main initialState) {
             // 3. Def1 registered, Def 2 unrepresented
             .transitionTo(PENDING_CLAIM_ISSUED_UNREPRESENTED_DEFENDANT)
                 .onlyIf((respondent1NotRepresented.and(respondent2NotRepresented))
+                    .or(respondent1NotRepresented.and(respondent2OrgNotRegistered.negate()))
+                    .or(respondent1OrgNotRegistered.negate().and(respondent2NotRepresented))
+                    .and(not(specClaim)))
+            .transitionTo(PENDING_CLAIM_ISSUED_UNREPRESENTED_DEFENDANT_ONE_V_ONE_SPEC)
+                .onlyIf(oneVsOneCase.and(respondent1NotRepresented).and(specClaim))
+            .transitionTo(PENDING_CLAIM_ISSUED_UNREPRESENTED_DEFENDANT)
```

Comment:
whats the difference between this and line 237?

Reply:
237 looks to be for when the case is multiparty and this is for a 1v1 case
Is that right @kenneth-hmcts

Reply:
The PENDING_CLAIM_ISSUED_UNREPRESENTED_DEFENDANT is also used in Damages team which is used in both multiparty and 1v1 case while LR just used for 1v1 case currently. We need to set up by adding multipartyCase and specClaim for that.